### ISSUE TITLE
Issues with light theme

### ISSUE THREAD

**Issue by Ranguna** [Frewacom/pywalfox#92]
When using pywal to generate a light theme and applying it in firefox, some text becomes unreadable:
The active tab:
![image](https://user-images.githubusercontent.com/5003755/184533169-380529db-7085-4721-9b67-c33ee94d6e14.png)

The extensions itself:
![image](https://user-images.githubusercontent.com/5003755/184533198-e9a27a58-a933-4016-b64f-3ea79335f17b.png)

The reason I want to explicitly use the `light` option though `wal` cli is because some tools like the terminal and vscode only change to the `light` option when generating the colors with that option.

Changing to the light option on the extensions makes things a bit more unreadable:
![image](https://user-images.githubusercontent.com/5003755/184533285-5d8bc825-6400-4c73-9b18-d0c718f7423f.png)

Is there any way to make this extensions compatible with colors generated with the `light` cli option ?

**Comment by Frewacom** [comment]
You can modify the template used for picking the colors, but that is not really a solution. I don't quite remember how the .cache/wal/colors.json file look - does it include anything that would indicate that it is a "light" theme? If not, I guess pywalfox could check the brightness of some colors to try ans figure out whether it is a light or dark mode.

**Comment by Ranguna** [comment]
Wow thanks for the fast reaction to this one!

As for whether colors.json indicates which color scheme is in use, unfortunately it does not :(
Here's my colors.json at the moment:
```json
{
    "wallpaper": "/path/to/my/image.png",
    "alpha": "100",

    "special": {
        "background": "#f9f9f9",
        "foreground": "#3e3e3e",
        "cursor": "#3e3e3e"
    },
    "colors": {
        "color0": "#f9f9f9",
        "color1": "#6e535d",
        "color2": "#93949f",
        "color3": "#bfa2ab",
        "color4": "#a8bcc8",
        "color5": "#ccabc3",
        "color6": "#cdbcb3",
        "color7": "#3e3e3e",
        "color8": "#bababa",
        "color9": "#6e535d",
        "color10": "#93949f",
        "color11": "#bfa2ab",
        "color12": "#a8bcc8",
        "color13": "#ccabc3",
        "color14": "#cdbcb3",
        "color15": "#3e3e3e"
    }
}
```

I also created https://github.com/dylanaraps/pywal/issues/685, with the hopes that maybe we could standardize this somehow, either by including both color schemes in colors.json or not recommending consumers to have a dark/light option in each application.

> If not, I guess pywalfox could check the brightness of some colors to try and figure out whether it is a light or dark theme.

Is there a reason you need to know which theme is active ?
The vscode extension and the terminal emulator theme simply uses the active theme and won't give you the option to toggle; if you want to change, you'll have to run the wal command with or without the `-l` flag.
So maybe the same could be done in pywalfox (if wal does not provide two color scheme in the colors.json): not having the option to choose between dark and light and simply use the colors in colors.json.

**Comment by Frewacom** [comment]
> Is there a reason you need to know which theme is active ?
I think the colors that should be used when theming the browser is different for light and dark mode. Using the same color indicies for light mode results in a very ugly looking theme, which is why I do some manual adjustments to the colors in the code when light mode is activated. I haven't looked at this in a long time, so I might be wrong though.

**Comment by Ranguna** [comment]
Makes sense, I can try taking a look at this in the future and check which colors the terminal and vscode use for text and focus text, thanks for all the info!

**Comment by holmescharles** [comment]
Any progress on this? I am experiencing the same issue.

**Comment by Packjackisback** [comment]
Yep, same thing, the only way I see now is forking it.

**Comment by Ranguna** [comment]
@Packjackisback or propose a PR

**Comment by holmescharles** [comment]
@Ranguna, this repo does not seem to be maintained anymore, unfortunately. So a fork would be best.

**Comment by Ranguna** [comment]
Hmmm maybe
@Frewacom are you still around?

**Comment by Ranguna** [comment]
> Yes, I am just busy with other stuff. PRs are appreciated. 

Good to know, thanks!

**Comment by johanengstrand** [comment]
We can continue this in #163.
